parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -923,6 +938,142 @@ impl_runtime_apis! {
 			type XcmBalances = pallet_xcm_benchmarks::fungible::Pallet::<Runtime>;
 			type XcmGeneric = pallet_xcm_benchmarks::generic::Pallet::<Runtime>;
 
+			use bridge_runtime_common::messages_benchmarking::{prepare_message_delivery_proof, prepare_message_proof};
+			use pallet_bridge_messages::benchmarking::{
+				Config as BridgeMessagesConfig,
+				Pallet as BridgeMessagesBench,
+				MessageDeliveryProofParams,
+				MessageProofParams,
+			};
+
+			impl BridgeMessagesConfig<WithBridgeHubWococoMessagesInstance> for Runtime {
+				// TODO: remove me after bridges subtree update
```

Comment:
@svyatonik 
now, we have very latest master from bridges repo here in subtree, so could this be removed?